### ISSUE TITLE
uninstall command?

### ISSUE THREAD

**Issue by tunnckoCore** [bpkg/bpkg#64]

**Comment by tunnckoCore** [comment]
bump?

**Comment by jwerle** [comment]
We'd happily accept a PR or a more detailed approach on how to implement
this

**Comment by jwerle** [comment]
possible duplicate of #31

**Comment by tunnckoCore** [comment]
@jwerle nope, his issue is for uninstalling package, i'm talking for uninstalling `bpkg` from the system.

**Comment by gutierri** [comment]
@tunnckoCore Referring to bpkg itself. What do you think?
`bpkg uninstall [-g] bpkg`

**Comment by tunnckoCore** [comment]
Maybe yes.

**Comment by hydra1983** [comment]
@olstenlarck GH-94 the ```uninstall``` command is supported now. but ```PREFIX``` is not working yet
